tests/loggers/test_all.py

```diff
@@ -365,6 +365,26 @@ def _test_logger_created_on_rank_zero_only(tmpdir, logger_class):
     assert trainer.state.finished, f"Training failed with {trainer.state}"
 
 
+@pytest.mark.parametrize(
+    "logger_value",
+    [
+        None,
+        False,
+        [],
+    ],
+)
+def test_no_logger(tmpdir, logger_value):
```

Comment:
this test would better fit into tests/trainer/flags. 
the tests here under test_all are meant to test the implementation of the logger api

Reply:
good point. fixed!